pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -967,4 +967,33 @@ rec {
       platforms = platforms.all;
     };
   };
+
+  /* ROMANIAN */
+  ro_RO = ro-ro;
+  ro-ro = mkDict rec {
+    pname = "hunspell-dict-ro-ro";
+    version = "3.3.10";
+    shortName = "ro-ro";
+    dictFileName = "ro_RO";
+    fileName = "${dictFileName}.${version}.zip";
+    shortDescription = "Romanian (Romania)";
+    readmeFile = "README";
+
+    src = fetchurl {
+      url = "https://downloads.sourceforge.net/rospell/${fileName}";
+      sha256 = "7f128d64ea06c9e6711c30b118c0afeefb014d8f33c92daccdf455aba2d04519";
+    };
+
+    nativeBuildInputs = [ unzip ];
+    unpackCmd = ''
+      unzip $src ${dictFileName}.aff ${dictFileName}.dic ${readmeFile} -d ${dictFileName}
+    '';
+
+    meta = with lib; {
+      description = "Hunspell dictionary for ${shortDescription} from rospell";
+      homepage = "https://sourceforge.net/projects/rospell/";
+      license = with lib.licenses; [ gpl2 lgpl21 mpl11 ];
+      #maintainers = with maintainers; [ Andy3153 ];
```

Comment:
You could also remove this line if you don't plan to add yourself as maintainer.

Reply:
I added that at first but then commented it out because I was unsure whether `maintainers` means package maintainers (me in this case) or maintainer of the program (where my package downloads from). If it means the first one, I'll uncomment that line.

Reply:
First one. You need to add yourself as [maintainers list](https://github.com/NixOS/nixpkgs/tree/master/maintainers) if you already haven't in order to add yourself as package maintainer.

Reply:
Done: b6c993a and #315849